### PR TITLE
config: Overrides ILS and circulation config

### PR DESCRIPTION
**Pull request by jrcastro2**
Depends on #https://github.com/inveniosoftware/invenio-app-ils/pull/960

### CODE REVIEW

File: cds_ils/circulation/utils.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+#
+# CDS-ILS is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+"""Circulation configuration callbacks."""
+
+
+def circulation_cds_extension_max_count(loan):
+    """Return a default extensions max count."""
+    unlimited = loan.get("extension_count", 0) + 1
```
Comment (kprzerwa) [inline cds_ils/circulation/utils.py:13]: nice way of defining infinity :D

**Review by kprzerwa — APPROVED** [review]
LGTM

**Comment by ntarocco** [comment]
please wait for https://github.com/CERNDocumentServer/cds-ils/pull/198 to be merged, there might be conflicts

File: cds_ils/templates/invenio_app_ils_circulation/mail/cds_active_loans.html [hunk 2]
```diff
@@ -5,7 +5,7 @@
 {% block body_plain %}
 Dear librarians,
 
-when removing users who are no longer registered in the system, it has been found a patron with active loans.
+when removing users who are no longer registered in the system, a patron with active loans has been found.
```
Comment (ntarocco) [inline cds_ils/templates/invenio_app_ils_circulation/mail/cds_active_loans.html:8]: I have completely rewritten this. If you remove it from your PR, you won't have a conflict.

File: ui/src/config.js [hunk 3]
```diff
@@ -24,6 +24,10 @@ export const config = {
       },
     },
   },
+  CIRCULATION: {
```
Comment (ntarocco) [inline ui/src/config.js:27]: great, did you check ALL config vars in `defaultConfig.js` in `react-invenio-app-ils`? I am wondering if there are more...

**Review by ntarocco — COMMENTED** [review]
If you push, please cancel the build until you rebase on master when the other PR is merged.

File: ui/src/config.js (hunk 3 above)
Comment (jrcastro2) [reply ui/src/config.js:27]: I double checked it and accordingly to what we discussed with the librarians these are the changes we need to do

**Review by ntarocco — APPROVED** [review]

**Pull request merged**